package-lock.json

```diff
@@ -1016,244 +1016,6 @@
 				"to-fast-properties": "^2.0.0"
 			}
 		},
-		"@bpmn-io/snarkdown": {
```

Comment:
Same as for the other component: https://github.com/bpmn-io/form-js/pull/255#issuecomment-1134578076

I think we can clean up this changes via `npm run reinstall`

Reply:
This is actually what happens after a cleanup. I cannot run reinstall on my machine without problems sadly but I deleted the package json + node modules and that's what I get out of it. I think there were some leftover non cleaned up items in the file.